backend/server.py

```
@@ -17,19 +25,74 @@
 from mediawikiapi import MediaWikiAPI
+from werkzeug.exceptions import RequestEntityTooLarge
+
+
+from multiprocessing import Process, Queue
+
+def execute_with_timeout(func, timeout, *args, **kwargs):
+    def wrapper(q, *args, **kwargs):
+        try:
+            result = func(*args, **kwargs)
+            q.put(("success", result))
+        except Exception as e:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)